src/services/egoTokenService.ts

```diff
@@ -73,7 +73,7 @@ export default function(egoURL: string = process.env.EGO_API, scope: string = pr
         try {
           const scopes = authToken['context']['scope'] as Array<string>;
           if (scopes.includes(scope)) {
-            const routePromise = fn(req, res, next);
+            const routePromise: any = fn(req, res, next);
```

Comment:
Unsure what type this could be

Reply:
I don't think it's a bid deal, but this is an express type `RequestHandler` example here https://github.com/overture-stack/ego-token-middleware/blob/master/src/index.ts#L2